docs/releases/jijmodeling-1.9.0.ipynb

```diff
@@ -0,0 +1,360 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "04beff77-c455-41ed-becd-84c5a672b01b",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": "slide"
+    },
+    "tags": []
+   },
+   "source": [
+    "# JijModeling 1.9.0 Release Note"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "648529c3-c8f5-4244-8840-d0585878f39d",
+   "metadata": {
+    "editable": true,
+    "execution": {
+     "iopub.execute_input": "2024-11-04T23:58:09.289297Z",
+     "iopub.status.busy": "2024-11-04T23:58:09.289099Z",
+     "iopub.status.idle": "2024-11-04T23:58:20.086704Z",
+     "shell.execute_reply": "2024-11-04T23:58:20.086414Z"
+    },
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import jijmodeling as jm"
+   ]
+  },
```

Comment:
Please merge the following Python import statements into the block located directly under the 'Example' section:

```python
import jijmodeling as jm
```

Reply:
Fixed in fd181390ac8c70751738fdb0ef0a5da59b82dcf5.